src/krotov/objectives.py

```diff
@@ -15,6 +15,11 @@
 
 import numpy as np
 import qutip
+from packaging.version import parse as parse_version
+if parse_version(qutip.__version__) < parse_version("5"):
+    is_qutip5 = False
```

Comment:
This is going to go into `krotov v2.0`, and I think it would be fine to make `qutip v5.0` a requirement for that release. So I don't think this is needed. We can drop compatibility with QuTiP 4